.github/workflows/build-samples.yml

```diff
@@ -31,6 +33,49 @@ jobs:
         run: |
           ./apko build ./examples/nginx.yaml nginx:build /tmp/nginx-${{ matrix.arch }}.tar --debug --arch ${{ matrix.arch }}
 
+      - name: Check SBOM Conformance
+        run: |
+          set -euxo pipefail
+          for f in *.spdx.json; do
+            echo ::group::sbom.json
+            cat $f
+            echo ::endgroup::
+            docker run --rm -v $(pwd)/$f:/sbom.json cgr.dev/chainguard/ntia-conformance-checker -v --file /sbom.json
+          done
+
+  # Build a multi-arch nginx image for all archs.
+  build-nginx-multiarch:
+    name: build-nginx-multiarch
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v2.1.5
+        with:
+          go-version-file: 'go.mod'
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@68827325e0b33c7199eb31dd4e31fbe9023e06e3 # v3.0.0
+
+      - name: build
+        run: |
+          make apko
+          ./apko version
```

Comment:
Why do we need this?

Reply:
Just a sanity check. I copied it from the other one.